packages/manager/modules/anthos/src/dashboard/general-information/general-information.html

```diff
@@ -0,0 +1,235 @@
+    </oui-header>
+
+    <div class="row py-4">
+
+        <!--General information tile-->
+        <div class="col-md-4">
+            <oui-tile
+                data-heading="{{:: 'anthos_tenant_dashboard_general_information_tile_information_title' | translate }}"
+            >
+                <!--Description-->
+                <oui-tile-definition
+                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_information_description' | translate }}"
+                    data-description="{{:: $ctrl.tenant.description }}"
+                >
+                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
+                        <oui-action-menu-item data-on-click="$ctrl.goToRenameService()">
+                            <span
+                                data-translate="anthos_tenant_dashboard_general_information_tile_information_description_action_rename"
+                            ></span>
+                        </oui-action-menu-item>
+                    </oui-action-menu>
+                </oui-tile-definition>
+
+                <!--Software-->
+                <oui-tile-definition
+                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_information_software' | translate }}">
+                    <oui-tile-description>
+                        <p class="mb-0" data-ng-bind=":: $ctrl.tenant.version"></p>
+                        <p class="oui-badge oui-badge_warning"
+                           data-translate="anthos_tenant_dashboard_general_information_tile_information_software_status_upcoming">
+                        </p>
+                    </oui-tile-description>
+                </oui-tile-definition>
+
+                <!--Location-->
+                <oui-tile-definition
+                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_information_location' | translate }}"
+                    data-description="{{:: $ctrl.tenant.location}}"
+                >
+                </oui-tile-definition>
+            </oui-tile>
+        </div>
+
+        <!--Security and access-->
+        <div class="col-md-4">
+            <oui-tile
+                data-heading="{{:: 'anthos_tenant_dashboard_general_information_tile_security_title' | translate }}"
+            >
+                <!--Anthos console-->
+                <oui-tile-definition
+                    data-term="{{:: 'anthos_tenant_dashboard_general_information_tile_security_center_access_anthos_title' | translate }}">
+
+                    <oui-tile-description>
+                        <oui-clipboard class="mb-2" data-model="$ctrl.tenant.accessUrl"></oui-clipboard>
+                        <oui-button
+                            data-size="m"
+                            data-icon-right="oui-icon-external-link"
+                            data-on-click="$ctrl.onOpenAnthosConsoleClick()">
```

Comment:
Can use `openAnthosConsole()` directly here

Reply:
Related to this:
https://github.com/ovh/manager/pull/5433#discussion_r693757990